contributing.md

```diff
@@ -95,6 +97,39 @@ We're a UNIX-friendly organization; we love making lots of small modules that do
 
 If you're creating a new repository, please follow the new repository guidelines [orbitdb/repo-template](https://github.com/orbitdb/repo-template).
 
+#### Linting
+
+To add [`standard`](https://standardjs.com/) to a repository:
+
+```sh
+npm i -D standard # npm install --save-dev standard
+```
+
+Manually add `"lint": "standard --env=mocha"` to your `package.json` `.scripts` field. Then, run `standard --fix` to fix current issues, and `npm run lint` to show the output of the linter.
+
+If you'd rather have the environment set permanently (for instance, so that your editor can notice it), add this to the `package.json`:
+
+```json
+{
+  "standard": {
+    "env": [
+      "mocha"
+    ]
+  }
+}
+```
+
+Extra-credit: If you really want to speed this process up, run `jq '.scripts.lint="standard --env=mocha"' package.json | sponge package.json` to automatically install standard, using [`jq`](https://stedolan.github.io/jq/) and `sponge` from GNU's `moreutils`.
```

Comment:
For extra credit, install [`jq`](https://stedolan.github.io/jq/) and `sponge` from GNU's `moreutils` and do it in one line:
```
$`jq '.scripts.lint="standard --env=mocha"' package.json | sponge package.json`
```